inc/class-wpseo-replace-var-data.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Internals
+ * @since   7.0
+ */
+
+/**
+ * Class WPSEO_Replace_Var_Data
+ *
+ * This class is used by class-wpseo-replace-vars to store the data of a single row of the snippet variables table.
+ */
+class WPSEO_Replace_Var_Data {
```

Comment:
`Data` is a bit vague. I would choose for `WPSEO_Replacement_Variable`. That's what it is.